{"uuid":"674902f8-d74d-49ee-8d39-58e8f6d85322","historyId":"2a3641f6899f40fde367a0b0d30a5f26","fullName":"died_procesos_generales.pgForgotPassTest.ForgotPassTest.ForgotPassCaseTest02","labels":[{"name":"package","value":"died_procesos_generales.pgForgotPassTest.ForgotPassTest"},{"name":"testClass","value":"died_procesos_generales.pgForgotPassTest.ForgotPassTest"},{"name":"testMethod","value":"ForgotPassCaseTest02"},{"name":"parentSuite","value":"Login"},{"name":"suite","value":"Test_03"},{"name":"subSuite","value":"died_procesos_generales.pgForgotPassTest.ForgotPassTest"},{"name":"host","value":"MED00143N010"},{"name":"thread","value":"7028@MED00143N010.main(1)"},{"name":"epic","value":"Regression Tests"},{"name":"feature","value":"Forgot Tests"}],"links":[],"name":"ForgotPassCaseTest02","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [Por favor revisar el código de validación enviado a su celular, puede que tarde unos minutos en llegarle.] but found [Usted tiene una petición generada, espere 4 minutos e intente de nuevo.]","trace":"java.lang.AssertionError: expected [Por favor revisar el código de validación enviado a su celular, puede que tarde unos minutos en llegarle.] but found [Usted tiene una petición generada, espere 4 minutos e intente de nuevo.]\r\n\tat org.testng.Assert.fail(Assert.java:93)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:512)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:134)\r\n\tat org.testng.Assert.assertEquals(Assert.java:115)\r\n\tat org.testng.Assert.assertEquals(Assert.java:189)\r\n\tat org.testng.Assert.assertEquals(Assert.java:199)\r\n\tat died_procesos_generales.pgForgotPassPage.ForgotPassPage.verifyMessage01_aroundBody18(ForgotPassPage.java:245)\r\n\tat died_procesos_generales.pgForgotPassPage.ForgotPassPage$AjcClosure19.run(ForgotPassPage.java:1)\r\n\tat org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)\r\n\tat io.qameta.allure.aspects.StepsAspects.step(StepsAspects.java:46)\r\n\tat died_procesos_generales.pgForgotPassPage.ForgotPassPage.verifyMessage01(ForgotPassPage.java:245)\r\n\tat died_procesos_generales.pgForgotPassTest.ForgotPassTest.ForgotPassCaseTest02(ForgotPassTest.java:86)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)\r\n\tat java.base/java.lang.reflect.Method.invoke(Unknown Source)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:661)\r\n\tat org.testng.internal.Invoker.retryFailed(Invoker.java:1014)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:744)\r\n\tat org.testng.TestRunner.run(TestRunner.java:602)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:380)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:289)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1226)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1144)\r\n\tat org.testng.TestNG.run(TestNG.java:1115)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","start":1550160446898,"stop":1550160456360,"steps":[{"name":"Open N12 Step...","status":"passed","stage":"finished","start":1550160446898,"stop":1550160447920},{"name":"metodo de recuperar contraseña con celular","status":"passed","stage":"finished","start":1550160448420,"stop":1550160454365,"parameters":[{"name":"row","value":"<xml-fragment r=\"12\" spans=\"1:14\" x14ac:dyDescent=\"0.25\" xmlns:mc=\"http://schemas.openxmlformats.org/markup-compatibility/2006\" xmlns:r=\"http://schemas.openxmlformats.org/officeDocument/2006/relationships\" xmlns:x14ac=\"http://schemas.microsoft.com/office/spreadsheetml/2009/9/ac\" xmlns:main=\"http://schemas.openxmlformats.org/spreadsheetml/2006/main\">\r\n  <main:c r=\"A12\" s=\"5\">\r\n    <main:v>2</main:v>\r\n  </main:c>\r\n  <main:c r=\"B12\" s=\"16\" t=\"s\">\r\n    <main:v>51</main:v>\r\n  </main:c>\r\n  <main:c r=\"C12\" s=\"10\" t=\"s\">\r\n    <main:v>63</main:v>\r\n  </main:c>\r\n  <main:c r=\"D12\" s=\"10\" t=\"s\">\r\n    <main:v>32</main:v>\r\n  </main:c>\r\n  <main:c r=\"E12\" s=\"2\"/>\r\n  <main:c r=\"F12\" s=\"2\" t=\"s\">\r\n    <main:v>36</main:v>\r\n  </main:c>\r\n  <main:c r=\"G12\" s=\"2\"/>\r\n  <main:c r=\"H12\" s=\"2\" t=\"s\">\r\n    <main:v>31</main:v>\r\n  </main:c>\r\n  <main:c r=\"I12\" s=\"1\" t=\"s\">\r\n    <main:v>18</main:v>\r\n  </main:c>\r\n  <main:c r=\"J12\" s=\"2\" t=\"s\">\r\n    <main:v>38</main:v>\r\n  </main:c>\r\n  <main:c r=\"K12\" s=\"2\" t=\"s\">\r\n    <main:v>35</main:v>\r\n  </main:c>\r\n  <main:c r=\"L12\" s=\"1\"/>\r\n  <main:c r=\"M12\" s=\"1\"/>\r\n  <main:c r=\"N12\" s=\"1\"/>\r\n</xml-fragment>"}]},{"name":"Verify username: {0} step...","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [Por favor revisar el código de validación enviado a su celular, puede que tarde unos minutos en llegarle.] but found [Usted tiene una petición generada, espere 4 minutos e intente de nuevo.]","trace":"java.lang.AssertionError: expected [Por favor revisar el código de validación enviado a su celular, puede que tarde unos minutos en llegarle.] but found [Usted tiene una petición generada, espere 4 minutos e intente de nuevo.]\r\n\tat org.testng.Assert.fail(Assert.java:93)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:512)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:134)\r\n\tat org.testng.Assert.assertEquals(Assert.java:115)\r\n\tat org.testng.Assert.assertEquals(Assert.java:189)\r\n\tat org.testng.Assert.assertEquals(Assert.java:199)\r\n\tat died_procesos_generales.pgForgotPassPage.ForgotPassPage.verifyMessage01_aroundBody18(ForgotPassPage.java:245)\r\n\tat died_procesos_generales.pgForgotPassPage.ForgotPassPage$AjcClosure19.run(ForgotPassPage.java:1)\r\n\tat org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)\r\n\tat io.qameta.allure.aspects.StepsAspects.step(StepsAspects.java:46)\r\n\tat died_procesos_generales.pgForgotPassPage.ForgotPassPage.verifyMessage01(ForgotPassPage.java:245)\r\n\tat died_procesos_generales.pgForgotPassTest.ForgotPassTest.ForgotPassCaseTest02(ForgotPassTest.java:86)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)\r\n\tat java.base/java.lang.reflect.Method.invoke(Unknown Source)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:661)\r\n\tat org.testng.internal.Invoker.retryFailed(Invoker.java:1014)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:744)\r\n\tat org.testng.TestRunner.run(TestRunner.java:602)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:380)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:289)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1226)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1144)\r\n\tat org.testng.TestNG.run(TestNG.java:1115)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","start":1550160454867,"stop":1550160454929,"parameters":[{"name":"expectedUserNameMessage","value":"Por favor revisar el código de validación enviado a su celular, puede que tarde unos minutos en llegarle."}]}],"attachments":[{"name":"Page screenshot","source":"eef73de7-6787-4a63-a2eb-44b2c1d088db-attachment","type":"image/png"},{"name":"{0}","source":"87c7c4cd-4bf6-4e40-8613-1b03b9989a06-attachment","type":"text/plain"},{"name":"Page screenshot","source":"b6bb9f0b-428c-423f-95c7-a235161c08f6-attachment","type":"image/png"},{"name":"{0}","source":"e972272d-5f0d-49d9-9a9b-25b0adf6b64d-attachment","type":"text/plain"}],"parameters":[]}